### ISSUE TITLE
[Wishlist] Able to use company-mode for yas-prompt-functions

### ISSUE THREAD

**Issue by bandresen** [joaotavora/yasnippet#836]
Hello!

Right now I'm using `yas-completing-prompt` and it works functionality wise but I would like it more if it was in place, like company mode is and not in the minibuffer.

Thanks in advance,
benny

**Comment by npostavs** [comment]
Have you tried `yas-x-prompt`? Or `yas-dropdown-prompt` (requires installing [dropdown-list](https://www.emacswiki.org/emacs/dropdown-list.el))? I think the latter is most like company-mode.

**Comment by bandresen** [comment]
The benefit to company mode would be uniformity which is not present with `yas-x-prompt` nor with `yas-dropdown-prompt`.

So that's a no that company-mode will be used by yasnippet?

**Comment by npostavs** [comment]
> So that's a no that company-mode will be used by yasnippet?

I'm not sure what that means. Does [company-yasnippet](https://github.com/company-mode/company-mode/blob/master/company-yasnippet.el) help?

**Comment by bandresen** [comment]
No, it's purely about the `yas-prompt'. Basically what yas-dropdown-prompt does except it uses company and not dropdown-list.

**Comment by npostavs** [comment]
Hmm, well I don't know how to use company like that. I would accept a patch for it.
